cibuildwheel/linux.py

```diff
@@ -272,7 +275,7 @@ def build(options: BuildOptions) -> None:
             # copy the output back into the host
             call(['docker', 'cp',
                   container_name + ':/output/.',
-                  os.path.abspath(options.output_dir)])
+                  str(options.output_dir.resolve())])
```

Comment:
Is the `str(...)` here to satisfy the type-checker, or is it required by `subprocess.call`?